iterator_practice.cpp

```diff
@@ -1,16 +1,13 @@
 #include <iostream>
-#include <iterator> // for iterator
 #include <vector> // for vectors 
 
-using namespace std;
```

Comment:
In spectre, we avoid `using namespace`. Instead, we write things out, so it's always clear if you're using something from std, as opposed to something we wrote